app/src/androidTest/java/com/hedvig/app/feature/marketpicker/screens/MarketSelectedScreen.kt

```diff
@@ -0,0 +1,15 @@
+package com.hedvig.app.feature.marketpicker.screens
+
+import com.agoda.kakao.text.KButton
+import com.hedvig.app.R
+import com.hedvig.app.feature.marketpicker.MarketSelectedFragment
+import com.kaspersky.kaspresso.screens.KScreen
+
+class MarketSelectedScreen : KScreen<MarketSelectedScreen>() {
```

Comment:
```suggestion
object MarketSelectedScreen : KScreen<MarketSelectedScreen>() {
```

with that, you can use it as such: `MarketSelectedScreen { // ...` instead of having to use `onScreen<MarketSelectedScreen> { // ...` 😉